R/rd.R

```diff
@@ -41,16 +41,16 @@ roclet_tags.roclet_rd <- function(x) {
     backref = tag_value,
     concept = tag_markdown,
     describeIn = tag_name_description,
-    description = tag_markdown,
-    details = tag_markdown,
+    description = tag_markdown_with_sections,
+    details = tag_markdown_with_sections,
     docType = tag_name,
     encoding = tag_value,
     evalRd = tag_code,
     example = tag_value,
     examples = tag_examples,
     family = tag_value,
     field = tag_name_description,
-    format = tag_markdown,
+    format = tag_markdown_with_sections,
```

Comment:
I think we should only allow in `@description` and `@details` for simplicity

Reply:
OK.

Reply:
done.